### PR TITLE
Adding info for Ludia Adhikary

### PR DESCRIPTION
**Pull request by LUDIAA**
This is my introduction.

### CODE REVIEW

File: student_info/ludia_adhikary.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+Hello, 
+My name is Ludia. I am a summer research student at CCA. I am a AstroCom student. 
+I like to play soccer and watch soccer. I like to eat, travel and sleep.
```
Comment (Angela6884) [inline student_info/ludia_adhikary.csv:4]: That's cool Ludia!! I'm not into sports but soccer does look exciting.

Comment (Angela6884) [reply student_info/ludia_adhikary.csv:4]: Also do you play badminton?

Comment (nai-bby) [inline student_info/ludia_adhikary.csv:4]: What is your favorite food?

**Review by Angela6884 — APPROVED** [review]
Soccer is a cool sport, dude. LGTM

**Review by nai-bby — APPROVED** [review]
LGTM

**Pull request merged**